pkgs/development/python-modules/cpycppyy/default.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  cppyy-cling,
+  cppyy-backend,
+  cmake,
+  buildPythonPackage,
+  setuptools,
+  python3,
+  wheel,
+  pip,
+  stdenv,
+}:
+
+buildPythonPackage rec {
+  pname = "CPyCppyy";
+  version = "1.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wlav";
+    repo = pname;
+    rev = "8bd9d9e10d658a33294a4ba9b2251bf1f9223469";
+    hash = "sha256-IEwdJSXG6diz7j2N7WQys5na7+xj1CYu7J3FCkW9/AQ=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+    pip
+    cmake
+    cppyy-cling
+    cppyy-backend
+  ];
+
+  propagatedBuildInputs = [
+    cppyy-cling
+    cppyy-backend
+  ];
+
+  patches = [
+    ./cpycppy-setup.patch
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  configurePhase = ''
+    runHook preConfigure
+    sed -i "s|%cflags_placeholder%|return '${lib.optionalString stdenv.hostPlatform.isLinux "-pthread"} -std=c++2a'|" setup.py
+    runHook postConfigure
+  '';
+
+  pipInstallFlags = [
+    "--no-use-pep517"
+    "--no-deps"
+  ];
+
+  postInstall = ''
+    cmake ./
+    make
+    cp libcppyy.so $out/${python3.sitePackages}/
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times.